Makefile

```diff
@@ -72,7 +72,8 @@ GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
 DOCKER_IMAGE := docker-dev$(if $(GIT_BRANCH_CLEAN),:$(GIT_BRANCH_CLEAN))
 DOCKER_PORT_FORWARD := $(if $(DOCKER_PORT),-p "$(DOCKER_PORT)",)
 
-DOCKER_FLAGS := docker run --rm -i --privileged $(DOCKER_CONTAINER_NAME) $(DOCKER_ENVS) $(DOCKER_MOUNT) $(DOCKER_PORT_FORWARD)
+#TODO: This -v flag needs to be moved
+DOCKER_FLAGS := docker run -v /var/tmp:/tmp --rm -i --privileged $(DOCKER_CONTAINER_NAME) $(DOCKER_ENVS) $(DOCKER_MOUNT) $(DOCKER_PORT_FORWARD)
```

Comment:
This is necessary because we need to provide a file path which is present in a mounted volume inside the docker container, otherwise the function file.stat() provides different values for device and inode. This way we get the accurate system level values for device and inode of the given file.